docs/markdown/Reference-manual.md

```diff
@@ -2134,6 +2134,9 @@ are immutable, all operations return their results as a new string.
 - `join(list_of_strings)`: the opposite of split, for example
   `'.'.join(['a', 'b', 'c']` yields `'a.b.c'`.
 
+- `replace(old_char, new_char)`: replaces instances of `old_char` in the string
+  with `new_char`
```

Comment:
We should make it clear that it's not an inplace replace, but return a new string.

Reply:
I would also specify "replaces **all** instances".